src/adapters/Base/index.test.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable camelcase */
+import requestManager from 'lib/requestManager';
+
+import baseAdapter from '.';
+
+jest.mock('lib/requestManager');
+
+describe('BaseAdapter', () => {
+  const apiPath = '/sample';
+  const params = {
+    testKey: 'test value',
+  };
+
+  beforeEach(() => {
+    (requestManager as jest.Mock).mockImplementation(() => jest.fn());
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  describe('get', () => {
```

Comment:
Minor but you should capitalize those to emphasize as HTTP methods
```suggestion
  describe('GET', () => {
```